### PR TITLE
fix: Artikel Page & Tentang Kami Page

### PR DESCRIPTION
**Pull request by pockypoem**
## Describe your changes
I added a Title, Subtitle, and 4 filter buttons in the hero section on the Artikel page. Then, I implemented slider functionality and adjusted the responsiveness of the Artikel page. I also made some refinements to the hero section for the Tentang Kami page.

## JIRA / Trello ticket number and link
https://trello.com/c/SXu9rNac/5-fswuser-halaman-artikel

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added tests.

## Does your PR have breaking change?
No

## Test Video / Screenshots
ss-Artikel

### CODE REVIEW

File: src/pages/Artikel/index.tsx [hunk 1]
```diff
@@ -3,81 +3,245 @@ import Figure from "src/components/molecules/Figure";
...
 import Gambar from "src/assets/Artikel/hero-bg.jpg";
 import Carousel from "src/assets/Artikel/carousel-img.jpg";
-import Card from "src/assets/Artikel/card.jpg";
-// import ChevronLeft from "src/assets/ChevronLeft.svg";
 import ChevronRight from "src/assets/ChevronRight.svg";
 
 function Artikel() {
   const breadcrumbSteps = [{ text: "Artikel", link: "/artikel" }, { text: "" }];
 
+  const [activeFilter, setActiveFilter] = useState("Semua");
```
Comment (angga-22) [inline src/pages/Artikel/index.tsx:15]: @pockypoem better simpan default nilai sbgai konstanta

Comment (angga-22) [reply src/pages/Artikel/index.tsx:15]: useState<TFilter>("Semua")

const TFilter = "Semua" | "Sebagian" | "Sedikit"

File: src/pages/Artikel/index.tsx [hunk 2]
```diff
@@ -3,81 +3,245 @@ import Figure from "src/components/molecules/Figure";
...
+                onClick={() => handleFilterChange("Wisata")}
+              >
+                Wisata
+              </button>
+              <button
+                className={`filter-button ${
+                  activeFilter === "Kuliner" &&
+                  "active-filter bg-white bg-opacity-20 px-3 py-2 rounded-md border-solid border-2 border-white "
+                }`}
+                onClick={() => handleFilterChange("Kuliner")}
```
Comment (angga-22) [inline src/pages/Artikel/index.tsx:87]: simpan nilai constants di filter nya, 

export const FILTER_ARTICLE = {
TERBARU: 'TERBARU',
WISATA: 'WISATA',
KULINER: 'KULINER',
DAN SETERUSNYA
}

@pockypoem

Comment (angga-22) [reply src/pages/Artikel/index.tsx:87]: tinggal pake, 

import { FILTER_ARTICLE } from 'src/constants';

handleFilterChange(FILTER_ARTICLE.WISATA)

**Review by angga-22 — APPROVED** [review]
LGTM

**Review by ARRARIAKU2 — APPROVED** [review]

**Pull request merged**